vital/types/point.h

```diff
@@ -49,19 +49,21 @@ namespace kwiver {
 namespace vital {
 
 template < unsigned N, typename T >
-class VITAL_EXPORT point
+class VITAL_EXPORT point : protected Eigen::Matrix< T, N, 1 >
 {
 public:
   using vector_type = Eigen::Matrix< T, N, 1 >;
   using covariance_type = covariance_<N, float>;
 
-  point() { }
-  point(vector_type const& v) : m_value(v) { }
-
+  point() : vector_type{ vector_type::Zero() } {}
+  point( vector_type v, covariance_type c )
```

Comment:
The previous version did not require covariance in CTOR. Are there use cases where a point is created without a covariance?

Reply:
Yes. You are apparently missing the main feature of this PR, which is that all of the ctors for the base class (i.e. the `Eigen` point type) are inherited :slightly_smiling_face:.

Reply:
To wit, from the description: "... in order to also *inherit the constructors* for the same" (emphasis added). Which is done here:
https://github.com/Kitware/kwiver/blob/7284df04ff2fc8dfa4d34d9bdf155f6d78d3ebf3/vital/types/point.h#L63

(Note that this will include the `(vector_type const&)` ctor, because that is the base class's copy ctor. More interestingly, however, it includes construction for the appropriate number of scalars.)